types/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// TypeScript Version: 3.5
+
+import {Node, Data} from 'unist'
+
+declare namespace unistUtilRemovePosition {
+  interface NodeWithUndefinedPosition extends Node {
+    position: undefined
+  }
+
+  interface NodeWithoutPosition {
+    [key: string]: unknown
+    type: string
+    data?: Data
+  }
```

Comment:
:thinking: this would be equivalent to

```ts
interface NodeWithoutPosition extends Omit<Node, 'position'> {}
```

which would work across TS versions including v4.
I don't have a strong preference either way, but there are options here.

@Murderlon @wooorm @remcohaszing @BarryThePenguin thoughts? :thought_balloon:

Reply:
I prefer your suggestion. In case something in `Node` changes, it will be picked up here as well. I’m also surprised that this works but just `Omit<Node, 'position'>` didn’t.